### PR TITLE
[WIP] Add EVM-compatible chains - Learn EVM

### LINKED ISSUE #133: Create EIP forks for EVM-compatible chain
This https://github.com/crytic/building-secure-contracts/blob/master/learn_evm/eips_forks.md, but for BSC/Avalanche, etc..

### PR DESCRIPTION
**Pull request by suryansh-tob**
WIP Fix #133 
- Set up list associating every TIP to a TRON upgrade
- TODO for TRON: Shorten "What it does" and move other info to "Notes". Specify forced upgrade or not.
- TODO: Add other EVM-compatible chains to Learn EVM

### CODE REVIEW

**Comment by suryansh-tob** [comment]
- Set up for Celo, associating each CIP to a Celo fork
- Updated Learn EVM README to list Celo and TRON
- Set up for BSC, associating each BEP to a BSC fork (less info on this, WIP)

File: README.md [hunk 1]
```diff
@@ -12,18 +12,21 @@ Follow our guidelines and best practices to write secure smart contracts.
   - [Secure development workflow](./development-guidelines/workflow.md): A rough, high-level process to follow while you write code
   - [Token integration checklist](./development-guidelines/token_integration.md): What to check when interacting with arbitrary token
 - [Learn EVM](./learn_evm): EVM technical knowledge
-  - [EIPs - forks](./learn_evm/eips_forks.md): summarize the EIPs included in each fork
-- [Not so smart contracts](./not-so-smart-contracts): Examples of smart contract common issues. Each issue contains a description, an example and recommendations
+  - [Forks <> EIPs](./learn_evm/eips_forks.md): Summarize the EIPs included in each Ethereum fork
+    - [Forks <> CIPs](./learn_evm/cips_forks.md): Summarize the CIPs and EIPs included in each Celo fork *(EVM-compatible chain)*
+    - [Upgrades <> TIPs](./learn_evm/tips_upgrades.md): Summarize the TIPs included in each TRON upgrade *(EVM-compatible chain)*
```
Comment (montyly) [inline README.md:17]: We need to add binance smart chain here

File: learn_evm/cips_forks.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+[WIP] The following lists every CIP associated to a Celo fork. Celo is an EVM-compatible chain.
```
Comment (montyly) [inline learn_evm/cips_forks.md:1]: WIP can be removed right?

File: learn_evm/README.md [hunk 3]
```diff
@@ -3,3 +3,5 @@
 List of EVM technical knowledge 
 
 - [Forks <> EIPs](eips_forks.md): Summarize the EIPs included in each fork
+  - [Forks <> CIPs](cips_forks.md): Summarize the CIPs and EIPs included in each Celo fork (EVM-compatible chain)
+  - [Upgrades <> TIPs](tips_upgrades.md): Summarize the TIPs included in each TRON upgrade (EVM-compatible chain)
```
Comment (montyly) [inline learn_evm/README.md:7]: We need to add binance smart chain here too

**Review by montyly — COMMENTED** [review]
Looks great. We mostly just need to update the tables of content to include BSC

**Comment by suryansh-tob** [comment]
Linked to BEPs forks within parent READMEs (held off bc of lack of available info) and removed WIP tag for Celo.

**Pull request merged**